### ISSUE TITLE
Adapt Saturn 'close button' extension for JupyterLab

### ISSUE THREAD

**Issue by bradtaylor** [DataBiosphere/saturn-ui#662]
We are adding support for JupyterLab per XXXX.
We need the same 'close button' option as on standard Jupyter. Figure out how Lab loads extensions and pass a (modified if need be) version of our current extension

**Comment by zarsky-broad** [comment]
Blocked by https://github.com/DataBiosphere/leonardo/issues/595

**Comment by bradtaylor** [comment]
On hold for now, as JupyterLab is not ready for other reasons
